### PR TITLE
Feat/responsive

### PR DESCRIPTION
**Pull request by LukyVj**
🆕 ✨ 🍃

### CODE REVIEW

File: docgen/layouts/common/meta.pug [hunk 1]
```diff
@@ -7,22 +7,22 @@ html
     meta(content='width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no', name='viewport')
     link(rel='icon', href="assets/img/react-instant-favicon.png")
     meta(content='IE=edge,chrome=1', http-equiv='X-UA-Compatible')
-    meta(content='#', name='description')
+    meta(content='REACT-INSTANTSEARCH - Boost search on your React apps with Algolia', name='description')
```
Comment (vvo) [inline docgen/layouts/common/meta.pug]: Can you make it lower case?

Comment (LukyVj) [inline docgen/layouts/common/meta.pug]: sure

Comment (LukyVj) [inline docgen/layouts/common/meta.pug]: done :)

File: docgen/layouts/common/meta.pug [hunk 2]
```diff
@@ -7,22 +7,22 @@ html
     meta(content='width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no', name='viewport')
     link(rel='icon', href="assets/img/react-instant-favicon.png")
     meta(content='IE=edge,chrome=1', http-equiv='X-UA-Compatible')
-    meta(content='#', name='description')
+    meta(content='React-instantsearch - Boost search on your React apps with Algolia', name='description')
```
Comment (vvo) [inline docgen/layouts/common/meta.pug]: It's react-instantsearch anywhere, everywhere

Comment (LukyVj) [inline docgen/layouts/common/meta.pug]: so all lowercase

**Pull request merged**